packages/elements/src/components/DropdownSelect/index.tsx

```diff
@@ -3,13 +3,17 @@ import Select, { Option, SelectProps } from 'rc-select'
 import { CustomTagProps } from 'rc-select/lib/interface/generator'
 import { Field, FieldProps } from 'formik'
 import CustomTag from './custom-tag'
+import { dropdownSelectFieldValidateRequire } from '../../utils/validators'
+import { checkError } from '../../utils/form'
 
 export interface DropdownSelectProps {
```

Comment:
`export interface DropdownSelectProps & rcSelectProps`

You can use like this instead of nest to rcSelectProps